content/posts/getting_started_upstash.md

```diff
@@ -224,7 +224,7 @@ After setting the environment variables, you can test your Redis service by runn
 genezio local
 ```
 
-Open the testing page in your browser by navigating to {{< external-link link="https://app.genez.io/test-interface/local?port=8083" >}}`https://app.genez.io/test-interface/local?port=8083`{{< /external-link >}}.
+Open the testing page in your browser by navigating to {{< external-link link="http://localhost:8083/explore" >}}`http://localhost:8083/explore`{{< /external-link >}}.
```

Comment:
I think the visible link should be just localhost:8083/explore because i think if it has the http part the external link will not work